### PR TITLE
Feature/add active admin

### PR DESCRIPTION
**Pull request by andreorvalho**
this adds active admin as part of our setup

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    #     end
+    #   end
+
+    #   column do
+    #     panel "Info" do
+    #       para "Welcome to ActiveAdmin."
+    #     end
+    #   end
+    # end
+  end # content
```
Comment (andreorvalho) [inline app/admin/dashboard.rb]: comments to erase!

File: app/admin/wall.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ActiveAdmin.register Wall do
+  index do
+    selectable_column
+    id_column
+    actions
+  end
+
+  form do |f|
+    f.inputs "Wall Details" do
```
Comment (andreorvalho) [inline app/admin/wall.rb]: if there is no inputs is this block necessary?

**Comment by jonybravo** [comment]
why is pull review still saying that spec to admin user is inexistent ?
at section "Your branch must be test infected."

**Comment by andreorvalho** [comment]
So let's read the message in this image:
screen shot 2015-10-30 at 10 21 15 am

it says that: app/admin/admin_user.rb is missing tests and not the app/models/admin_user.rb

Besides that it lists all the new files that don't have any tests. If notice it's all files inside the admin folder and the new devise views created automatically by devise.

This is what I meant, that sometimes pull review, which is obviously an app and automated, is not always right. The reason for this is: When we use code that is from gems, which we trust is already tested there we dont need to test it ourselves.

What does this mean?

For example let's look at the app/admin/admin_user.rb file:

```ruby
ActiveAdmin.register AdminUser do
  permit_params :email, :password, :password_confirmation

  index do
    selectable_column
    id_column
    column :email
    column :created_at
    actions
  end

  filter :email

  form do |f|
    f.inputs "Admin Details" do
      f.input :email
      f.input :password
      f.input :password_confirmation
    end
    f.actions
  end
end
```

let's analyze function by function:

- permit_params on the second line is something defined by active_admin(how do I know this? I either google or know it already)

- index on line 4 is a function that receives a block, which is also defined by active_admin

and so on. Because all this functions are done on the gem and tested there we don't to test again, because we TRUST the gem's code.

Make's sense? So we just ignore that advice.

**Comment by andreorvalho** [comment]
All the other comments from Pull review you can ignore for now and let's move on.

But one last thing:

this kind of commit messages don't tells us anything:
screen shot 2015-10-30 at 10 33 04 am

why 1,2,3,4 or 5?

besides that most of those commit's were not doing specs anymore, it was corrections from Pull review.
In this case you have 2 solutions:

1. If you make a commit and then after you push it, you want to add more stuff to the same commit you can do ammend:

- git add <filename>
- git commit --amend -m "new message" or just git commit --amend and then the last message is shown and you can just that.

2. You can create a new commit but with meaningful messages.

**Pull request merged**